app/src/main/res/values/strings.xml

```diff
@@ -1,9 +1,9 @@
 <resources>
     <string name="app_name">FinancialPal</string>
-    <string name="AddIncomeBtn">Add Income</string>
-    <string name="amountLbl">Amount</string>
-    <string name="categoryLbl">Category</string>
-    <string name="AmountTxt">$</string>
-    <string name="CategoryTxt" />
-    <string name="SubmitBtn">Submit</string>
+    <string name="btn__add_income">Add Income</string>
+    <string name="label_amount">Amount</string>
+    <string name="label_category">Category</string>
+    <string name="text_amount">$</string>
+    <string name="text_category" />
+    <string name="btn_submit">Submit</string>
```

Comment:
Changed all strings to the proper \<how>_\<description> format

Reply:
Spelling full words is a good idea.  While abbreviation saves typing, you forget which abbreviation you use, which ends up taking more time.